src/settings.c

```diff
@@ -384,11 +353,10 @@ static void settings_write_callback(uint16_t sender_id, uint8_t len, uint8_t *ms
   /* Extract parameters from message:
    * 3 null terminated strings: section, setting and value
```

Comment:
Should update this to new behaviour: potentially 4 null temrinated strings

Reply:
Done.